packages/fxa-auth-server/lib/routes/utils/securityEvent.js

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { AccountEventsManager } = require  ('../../account-events');
+const { Container } = require('typedi');
+
+export function recordSecurityEvent(name, opts) {
+  const mgr = Container.get(AccountEventsManager);
+  if (mgr == null || typeof mgr.recordSecurityEvent !== 'function') {
+    return;
+  }
+
+
+
+
+  try {
+
+  if (opts.db == null) throw 'Missing DB';
+  if (!opts.request) throw 'Missing request';
+  if (!opts.request?.app) throw 'Missing app';
+  if (!opts.request?.app?.clientAddress) throw 'Missing ip';
+  if (!opts.request?.auth) throw 'Missing auth';
+  if (!opts.request?.auth?.credentials) throw 'Missing credentials'
```

Comment:
nit: indenting is off here. _/suspiciously side-eyes Prettier_

Reply:
Also, typically mildly suboptimal to throw a string, and we should throw `Error` objects instead.
https://eslint.org/docs/latest/rules/no-throw-literal

Reply:
These were actually just in place for debugging purposes. I've removed them. The downstream call handles these errors, but wasn't producing a stack trace, so I included this during dev. Thanks for the catch.